app/views/content/sandbox.html.erb

```diff
@@ -4,46 +4,44 @@
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-xl"><%= t('page_titles.sandbox') %></h1>
 
-    <p class='govuk-body'>
-      The Apply sandbox is a test environment which allows you to experience the service as a candidate, training provider and software vendor.
-    </p>
+    <p class="govuk-body">The Apply sandbox is a test environment which allows you to experience the service as a candidate, training provider and software vendor.</p>
 
-    <h2 class='govuk-heading-m'>You can:</h2>
+    <p class="govuk-body">You can:</p>
 
-    <ul class='govuk-list govuk-list--bullet'>
+    <ul class="govuk-list govuk-list--bullet">
       <li>sign in as a candidate and submit an application</li>
       <li>sign in as a training provider and process an application</li>
-      <li>test Apply’s <%= govuk_link_to 'API integrations', 'https://www.apply-for-teacher-training.service.gov.uk/api-docs' %></li>
+      <li>test Apply’s <%= govuk_link_to 'API integrations', api_docs_home_path %></li>
```

Comment:
Is there a reason why we hard code the URL for the API docs on the sandbox start page? @duncanjbrown

Reply:
Yes — we want providers to use the production docs as the source of truth, because that's ultimately where they're going to be connecting to. Not that production has ever differed from Sandbox, but it felt right.

Reply:
Ah, that makes sense. Will revert.